Added clarifying documentation and tips to Getting Started -> Quickstart docs page (#285)
motion-canvas/motion-canvas: packages/docs/docs/getting-started/quickstart.mdx

### Creating a new project

Navigate to whichever directory you would like your new animation folder 
to be placed.  

Run the following command in order to scaffold a new Motion Canvas project:

```bash
npm init @motion-canvas
```

Enter the name of your project (Ex] my-first-animation)

> Enter the project path (Ex] my-first-animation)

Reviewer: "Ex]" is a really peculiar way of denoting an example, can we change it to:
```suggestion
Enter the project path (e.g. `my-first-animation`)
```
?
Putting text around the ticks (`` ` ``) will make it look console-like, `like this`.

Reviewer 2: Support this. The first time I read this I was a bit confused about this notation.

:::

A new folder with the title of your project should appear in the directory 
that you selected.

> Next, change into your newly created project directory.
> (Ex] cd my-first-animation)

Reviewer: You can use a code block here:
````suggestion
Next, change into your newly created project directory. For example:
```bash
cd my-first-animation
```
````

It will make it look like a console:
```bash
cd my-first-animation
```

The scaffolding command will create several files for you, but for now we're
going to focus on `src/scenes/example.tsx`, which is where we can add our

> animations. Open `example.tsx` in a text editor (Ex] nano example.tsx), and replace all code in the

Reviewer 3: Not sure it's worth being opinionated about which editor...

Reviewer: It's just an example..............

Depending on the size of your browser, you may want to zoom the preview out by
scrolling down on your mouse wheel while hovering over the preview panel.

If you do not see the circle animation in the preview pane, try the following:

1. Open up a new terminal window

2. Navigate to the directory where you created your animation project (Ex] my-first-animation)

> 3. Run npm run serve

Reviewer 4: Probably should add ticks around this as well:
```
3. Run `npm run serve`
```

4. Copy and paste the new preview link with the new port number (Ex] http://localhost:9001/)

> in a new browser tab.

Reviewer 3: Most terminals these days let you Ctrl-click the URL.
